doc/internals/time-coding.rst

```diff
@@ -426,11 +439,11 @@ For encoding the process is more or less a reversal of the above, but we have to
         dtype="datetime64[s]",
     )
     orig_values = np.array(
-        [-2002 * 365 - 121, -366, 365, 2000 * 365 + 119], dtype="int64"
+        [-2002 * 365 - 121, -366, 365, 2000 * 365 + 119], dtype="float64"
     )
```

Comment:
If I am reading correctly, this redefined array is not used (and is not totally consistent with the `dates` and existing `units`), so I think we can delete it.

Reply:
Good chance that I wanted to compare like the example before, good to keep as is and add:

`np.testing.assert_array_equal(values, orig_values * 24)`

Reply:
The first element of `orig_values * 24` will need an hour added, since `dates` slightly changed.